docs/spec_draft.md

```diff
@@ -348,6 +349,83 @@ logical operation.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.batch_norm_inference
+
+### Semantics
+
+Normalizes the `operand` tensor across batch and spatial dimensions, for each
```

Comment:
Since we don't define what batch and spatial dimensions are, let's reformulate in a more precise fashion:

```
Normalizes the `operand` tensor across all dimensions except for the 
`feature_index` dimension and produces a `result` tensor.
```